turn_alerts/views.py

```diff
@@ -0,0 +1,128 @@
+import re
+
+from django.http import JsonResponse
+from prometheus_client import Counter
+from rest_framework import generics, permissions, status
+from rest_framework.response import Response
+
+from sidekick.models import Organization
+from turn_alerts.serializers import TurnOutboundSerializer, WhatsAppWebhookSerializer
+
+from .models import TurnActions
+from .tasks import start_turn_journey
+
+message_requests_total = Counter(
+    "turn_alerts_message_requests_total",
+    "Total number of message requests",
+    [
+        "fallback_channel",
+        "direction",
+        "message_type",
+    ],
+)
+
+
+event_count = Counter(
+    "turn_alerts_event_count",
+    "Number of events",
+    ["message_status", "error_code", "conversation_id", "conversation_type"],
```

Comment:
I don't think we'll ever aggregate on `conversation_id`?

What are the possible values for `conversation_type`?

Reply:
Is it possible to add fallback channel here?

Reply:
The only conversation type I've seen so far is "service" and I'm not sure what it means. We can have fallbackchannel. I'll add that.